config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -122,6 +122,48 @@ periodics:
   annotations:
     testgrid-dashboards: sig-node-kubelet
     testgrid-tab-name: node-kubelet-alpha
+- name: ci-kubernetes-node-kubelet-alpha-kubetest2
+  interval: 1h
+  decorate: true
+  extra_refs:
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
+  - org: kubernetes
+    repo: test-infra
+    base_ref: master
+    path_alias: k8s.io/test-infra
+  decoration_config:
+    timeout: 65m
+  annotations:
+    testgrid-dashboards: sig-node-kubelet
+    testgrid-tab-name: node-kubelet-alpha-kubetest2
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  spec:
+    containers:
+    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20211111-d096cb0c5f-experimental
+      env:
+      - name: GOPATH
+        value: /go
+      command:
+      - runner.sh
+      args:
+      - kubetest2
+      - noop
+      - --test=node
+      - --
+      - --repo-root=.
+      - --gcp-project-type=node-e2e-project
+      - --gcp-zone=us-west1-b
+      - --parallelism=8
+      - --focus-regex=\[NodeConformance\]|\[NodeAlphaFeature:.+\]|\[NodeAlphaFeature\]
+      - --skip-regex=\[Flaky\]|\[Serial\]
+      - --test-args=--feature-gates=AllAlpha=true,CSIMigration=false,RotateKubeletServerCertificate=false,InTreePluginGCEUnregister=false --kubelet-flags="--cgroups-per-qos=true --cgroup-root=/"
+      - --image-config-file=/home/prow/go/src/k8s.io/test-infra/jobs/e2e_node/image-config.yaml
+      - --runtime-config=api/all=true
```

Comment:
`--gcp-project-type` and `--runtime-config` would work after this PR gets merged into kubetest2 [ https://github.com/kubernetes-sigs/kubetest2/pull/175] and once the kubekins image is updated

Reply:
should we wait for it? PR is merged

Reply:
We're good to go now

Images updated: https://github.com/kubernetes/test-infra/commit/70cdbf024ac9ad7c3b04e2eac1620d6a55d98070